src/cpp/fastdds/topic/TopicImpl.cpp

```diff
@@ -41,6 +41,8 @@ TopicImpl::TopicImpl(
     , qos_(&qos == &TOPIC_QOS_DEFAULT ? participant_->get_default_topic_qos() : qos)
     , listener_(listen)
     , user_topic_(nullptr)
+    , handle_()
```

Comment:
This is not necessary. Topic inherits from DomainEntity which already has instance_handle_

Reply:
🤔 It was there before this PR. I will remove it.

Reply:
Done in cf3e98a